core/src/labels/labelMesh.cpp

```diff
@@ -13,4 +19,102 @@ void LabelMesh::addLabel(std::unique_ptr<Label> _label) {
     m_labels.push_back(std::move(_label));
 }
 
+void LabelMesh::loadQuadIndices() {
+    if (s_quadGeneration == s_validGeneration) {
+        glBindBuffer(GL_ELEMENT_ARRAY_BUFFER, s_quadIndexBuffer);
+
+    } else {
+        s_quadGeneration = s_validGeneration;
+
+        std::vector<GLushort> indices;
+        indices.reserve(maxVertices / 4 * 6);
+
+        for (size_t i = 0; i < maxVertices; i += 4) {
```

Comment:
This seems like stateless work we can do at initialization rather than in the render loop.